src/roles/httpd/templates/foreman-ssl-vhost.conf.j2

```diff
@@ -45,6 +61,27 @@
   ProxyPass /pulp/container/ {{ httpd_pulp_content_backend }}/pulp/container/
   ProxyPassReverse /pulp/container/ {{ httpd_pulp_content_backend }}/pulp/container/
 
+  <Location "/pulp/deb">
+    RequestHeader unset X-CLIENT-CERT
+    RequestHeader set X-CLIENT-CERT "%{SSL_CLIENT_CERT}s" env=SSL_CLIENT_CERT
+    ProxyPass {{ httpd_pulp_content_backend }}/pulp/content disablereuse=on timeout=600
+    ProxyPassReverse {{ httpd_pulp_content_backend }}/pulp/content
+  </Location>
+
+  <Location "/pulp/isos">
+    RequestHeader unset X-CLIENT-CERT
+    RequestHeader set X-CLIENT-CERT "%{SSL_CLIENT_CERT}s" env=SSL_CLIENT_CERT
+    ProxyPass {{ httpd_pulp_content_backend }}/pulp/content disablereuse=on timeout=600
+    ProxyPassReverse {{ httpd_pulp_content_backend }}/pulp/content
+  </Location>
+
+  <Location "/pulp/repos">
+    RequestHeader unset X-CLIENT-CERT
+    RequestHeader set X-CLIENT-CERT "%{SSL_CLIENT_CERT}s" env=SSL_CLIENT_CERT
+    ProxyPass {{ httpd_pulp_content_backend }}/pulp/content disablereuse=on timeout=600
+    ProxyPassReverse {{ httpd_pulp_content_backend }}/pulp/content
+  </Location>
```

Comment:
IIRC these are technically deprecated as they are the legacy Pulp 2 URLs. I'm wondering if we should pursue their removal with a deprecation.

In the short term, perhaps merge all content URLs with a single `LocationMatch` and a comment above them that `deb`, `isos` and `repos` are deprecated?